docs-site/docs/user/project/projects/manage-projects.md

```diff
@@ -75,7 +75,16 @@ To update the URL of the project:
 3. Select **Change project URL**
 
 ## View Project History
-<!-- TODO: Add steps to view project history -->
+
+Prerequisite:
+- You must have at least the Maintainer role for the project.
```

Comment:
Can you also add in, that the user must be a direct or inherited member from the project's ancestors. Shared groups (namespace group links) and their members cannot see the history

Reply:
Added in 5d6e652c3e18dae83fd0b12b8b61e8b5df717799